scripts/zkai_d64_commitment_consistency_method_probe.py

```
@@ -0,0 +1,560 @@
+    )
+    return manifest
+
+
+def activation_usage(reference: dict[str, Any]) -> dict[str, Any]:
+    clamped = [
+        max(-FIXTURE.ACTIVATION_CLAMP_Q8, min(FIXTURE.ACTIVATION_CLAMP_Q8, value))
+        for value in reference["gate_projection_q8"]
+    ]
+    indices = [value + FIXTURE.ACTIVATION_CLAMP_Q8 for value in clamped]
+    if min(indices) < 0 or max(indices) >= EXPECTED_ACTIVATION_TABLE_LEAVES:
+        raise CommitmentConsistencyProbeError("activation lookup index escaped table domain")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)